docs/DafnyRef/UserGuide.md

```diff
@@ -299,6 +299,7 @@ This list is not exhaustive but can definitely be useful to provide the next ste
  <br><br>`if X {`<br>&nbsp;&nbsp;`  ...`<br>`} else {`<br>&nbsp;&nbsp;`  ...`<br>`}`<br>`assert A;` | `if X {`<br>&nbsp;&nbsp;`  ...`<br>&nbsp;&nbsp;`  assert A;`<br>`} else {`<br>&nbsp;&nbsp;`  ...`<br>&nbsp;&nbsp;`  assert A;`<br>`}`<br>`assert A;`
  <br><br><br><br><br>`assert forall x :: Q(x);` | [`forall x`](#sec-forall-statement)<br>&nbsp;&nbsp;`  ensures Q(x)`<br>`{`<br>&nbsp;&nbsp;`  assert Q(x);`<br>`};`<br>` assert forall x :: Q(x);`
  <br><br><br><br><br>`assert forall x :: P(x) ==> Q(x);` | [`forall x | P(x)`](#sec-forall-statement)<br>&nbsp;&nbsp;`  ensures Q(x)`<br>`{`<br>&nbsp;&nbsp;`  assert Q(x);`<br>`};`<br>` assert forall x :: P(x) ==> Q(x);`
+ <br>`assert exists x | P(x) :: Q(x);`<br>`assert exists x | P(x) :: Q'(x);` | `if x :| P(x) {`<br>&nbsp;&nbsp;`  assert Q(x);`<br>&nbsp;&nbsp;`  assert Q'(x);`<br>`} else {`<br>&nbsp;&nbsp;`  assert false;`<br>`}`
```

Comment:
What's going on here in with all the `<br>`s?

Reply:
The `<br>` are there to try to align the left code with similar code on the right, for desktop browsers at least. That way, it becomes easier to see what was added. See the result here:
https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#sec-failing-asserts-special-cases

Reply:
Unfortunately yes, but that works for the Jekyll website, which is what we want.
![image](https://user-images.githubusercontent.com/3601079/160898541-0d664912-7ba6-4ff1-bddb-2551c11e8bba.png)